src/base/constants.py

```diff
@@ -43,4 +42,9 @@ def _IsDevAppServer():
 
 DEBUG = IS_DEV_APPSERVER
 
-TEMPLATE_DIR = os.path.sep.join([os.path.dirname(__file__), '..', 'templates'])
+template_dir = ('static' if os.environ.get('AUTHORIZATION_METHOD')
```

Comment:
It's done because when we are rendering static files, mostly they are in static/ so thats why. This is used in base code and integrated with jinja2 configs.